aten/src/THC/generic/THCTensor.cpp

```diff
@@ -452,9 +452,9 @@ void THCTensor_(squeeze)(THCState *state, THCTensor *self, THCTensor *src)
     THTensor_setStrideAtDim(self, 0, 1);
     ndim = 1;
   }
+#endif
   THTensor_resizeDim(self, ndim);
 }
-#endif
```

Comment:
Whoa, how did this ever compile.

Reply:
because that flag was off, so it didn't matter.